### PR TITLE
Retry when running Appveyor to work around failures.

### PR DESCRIPTION
**Pull request by ezyang**
Signed-off-by: Edward Z. Yang <ezyang@cs.stanford.edu>

### CODE REVIEW

**Comment by 23Skidoo** [comment]
Yes please.

File: appveyor-retry.cmd [hunk 1]
```diff
@@ -0,0 +1,20 @@
+@echo off
+rem initiate the retry number
+set retryNumber=0
+set maxRetries=500
```
Comment (23Skidoo) [inline appveyor-retry.cmd]: Seems a bit high.

Comment (ezyang) [inline appveyor-retry.cmd]: How about 5? (I literally just copy pasted the code from someone else)

Comment (ezyang) [inline appveyor-retry.cmd]: Oh, the original script has just 3; that seems good to me.

Comment (23Skidoo) [inline appveyor-retry.cmd]: Yep, somewhere in that ballpark. `travis_retry` retries three times by default.

**Pull request merged**